styles/style.css

```diff
@@ -20,7 +20,8 @@
     /* Values */
     --default-font-size: 16px;
 }
-
+@import url(https://fonts.googleapis.com/css?family=Josefin+Sans:100,200,300,regular,500,600,700,100italic,200italic,300italic,italic,500italic,600italic,700italic);
+@import url(https://fonts.googleapis.com/css?family=Lato:100,100italic,300,300italic,regular,italic,700,700italic,900,900italic);
 *,
```

Comment:
You dont need it because you have the same one at your HTML. Delete it and declare fonts correctly according to the google fonts guide